### DISCUSSION TITLE
Solution for pyinstaller --onefile for .exe?

### DISCUSSION THREAD

**Discussion by lillkiz86** [TomSchimansky/CustomTkinter#1700] [Q&A] [answered]
Has anyone found a solution for the pyinstaller --onefile option that excludes .json? I am currently using --onedir but it sucks not being able to have just a sleek executable. 

Any work arounds?

**Comment by Akascape** [comment, marked as answer]
@lillkiz86 Check this: https://github.com/TomSchimansky/CustomTkinter/discussions/939
